docs/security-privacy-questionnaire.md

```diff
@@ -3,12 +3,12 @@
 > 01.  What information does this feature expose,
 >      and for what purposes?
 
-Payment Link URI is exposed. This includes the scheme (e.g., upi, bitcoin, paypal) and potentially other parameters embedded in the URI, such as payee address, amount, and currency. This information is necessary to identify the payment method and initiate the transaction.
+Payment Link URI is exposed by the website, and will be received by payment clients. This includes the scheme (e.g., upi, bitcoin, paypal) and other parameters embedded in the URI, such as payee address, amount, and currency. This information is necessary to identify the payment method and prompt the user to initiate a transaction.
```

Comment:
On the one hand, I like how short this is written - it's straight to the point, which is great for quick reviews.

On the other hand, the reviewers that are reading this will have some questions: What are the payment clients, how does a user agent get a payment client, how many payment clients will be there, where are the payment clients coming from?

If one reads this paragraph naively, then users are leaking some potentially private information to an unknown number of unidentified 3rd party websites, which could be malicious trackers.

Please keep in mind that the point of this questionnaire is to describe how you think about the user security and privacy and to convince reviewers that you're not implementing a big privacy hole in the browser. 

Perhaps try to read through https://www.w3.org/TR/privacy-principles/#intro and see if that gives you an idea of what reviewers would be expecting.

Reply:
Thanks! Updated accordingly. Please let me know for further concerns!